Week-6/ex-1/p1.js

```diff
@@ -0,0 +1,35 @@
+
+//Max Sum Contiguous Subarray
+// The first and the only argument contains an integer array, 
+// A. Output Format: Return an integer representing the maximum possible sum of the contiguous subarray.
+// Constraints: 1 <= N <= 1e6 -1000 <= A[i] <= 1000
+// For example
+
+/*Input 1: A = [1, 2, 3, 4, -10]  Output 1: 10
+Explanation 1: The subarray [1, 2, 3, 4] has the maximum possible sum of 10.
+// Input 2: A = [-2, 1, -3, 4, -1, 2, 1, -5, 4] Output 2: 6
+Explanation 2: The subarray [4,-1,2,1] has the maximum possible sum of 6.
+*/
+
+const A = [1, 2, 3, 4, -10];// 10
+// const A = [-2, 1, -3, 4, -1, 2, 1, -5, 4]; // 6
+
+let max_val = Number.MIN_VALUE;
+let max_end = 0;
```

Comment:
We don't define variables like max_val. We do it as maxVal. This is in Python. You've created it as Python.

Reply:
I will surly next time look into it.